include/nbl/asset/IDescriptorSetLayout.h

```diff
@@ -15,26 +15,29 @@ namespace asset
 {
 
 // TODO: move this to appropriate class
-enum E_DESCRIPTOR_TYPE : uint32_t
+enum E_DESCRIPTOR_TYPE : uint8_t
 {
-    EDT_COMBINED_IMAGE_SAMPLER = 1,
-    EDT_STORAGE_IMAGE = 3,
-    EDT_UNIFORM_TEXEL_BUFFER = 4,
-    EDT_STORAGE_TEXEL_BUFFER = 5,
-    EDT_UNIFORM_BUFFER = 6,
-    EDT_STORAGE_BUFFER = 7,
-    EDT_UNIFORM_BUFFER_DYNAMIC = 8,
-    EDT_STORAGE_BUFFER_DYNAMIC = 9,
-    EDT_INPUT_ATTACHMENT = 10,
-	// Provided by VK_EXT_inline_uniform_block
-	EDT_INLINE_UNIFORM_BLOCK_EXT = 1000138000,
+    EDT_COMBINED_IMAGE_SAMPLER = 0,
+    EDT_STORAGE_IMAGE,
+    EDT_UNIFORM_TEXEL_BUFFER,
+    EDT_STORAGE_TEXEL_BUFFER,
+    EDT_UNIFORM_BUFFER,
+    EDT_STORAGE_BUFFER,
+    EDT_UNIFORM_BUFFER_DYNAMIC,
+    EDT_STORAGE_BUFFER_DYNAMIC,
+    EDT_INPUT_ATTACHMENT,
 	// Provided by VK_KHR_acceleration_structure
-	EDT_ACCELERATION_STRUCTURE_KHR = 1000150000,
+	EDT_ACCELERATION_STRUCTURE,
+
+	// Support for the following is not available:
+	// Provided by VK_EXT_inline_uniform_block
+	// EDT_INLINE_UNIFORM_BLOCK_EXT,
 	// Provided by VK_NV_ray_tracing
-	EDT_ACCELERATION_STRUCTURE_NV = 1000165000,
+	// EDT_ACCELERATION_STRUCTURE_NV = 1000165000,
 	// Provided by VK_VALVE_mutable_descriptor_type
-	EDT_MUTABLE_VALVE = 1000351000,
-	EDT_INVALID = ~0u
+	// EDT_MUTABLE_VALVE = 1000351000,
+
+	EDT_COUNT
 };
 
 //! Interface class for Descriptor Set Layouts
```

Comment:
remove the C++ code example, its just confusing and outdated by now

Reply:
Resolved.